pepper-apis/src/main/resources/db-changes/schema/baseline-activity-instance.xml

```diff
@@ -18,9 +18,7 @@
             <column name="participant_id" type="bigint">
                 <constraints nullable="false" references="user(user_id)" foreignKeyName="activity_instance_participant_user_fk"/>
             </column>
-            <column name="is_readonly" type="boolean">
-                <constraints nullable="false"/>
-            </column>
+            <column name="is_readonly" type="boolean"/>
```

Comment:
Liquibase files that has already been executed are effectively set-in-stone. Liquibase will complain if we change it since the checksum of the file changed. Please revert this and make the change in a new liquibase file.

Reply:
Now that's hilarious...screwing up stuff I've done a hundred times. The consequences of changing something with a headache. Fixed.

Reply:
That's not good -- take care! 🛌 💪